tests/puppeteer/campaigns/sanity/02_productsBO/02_CRUDStandardProductInBO.js

```diff
@@ -20,14 +18,12 @@ let editedProductData;
 
 // creating pages objects in a function
 const init = async function () {
```

Comment:
I would completly delete this function and directly use initBasicPages with an object as an argument